conans/client/tools/oss.py

```diff
@@ -221,10 +224,55 @@ def get_solaris_version_name(version):
         elif version.minor() == "5.11":
             return "Solaris 11"
 
+    @staticmethod
+    def bash_path():
+        if "CONAN_BASH_PATH" not in os.environ:
+            bash_path = None
+            for ext in [".bat", ".exe", ".sh", ""]:
+                bash_path = bash_path or which("bash%s" % ext)
+        else:
+            bash_path = os.getenv("CONAN_BASH_PATH")
```

Comment:
I prefer other flow, first doing os.getenv, if True => return it

Reply:
ok. good